### DISCUSSION TITLE
How do I set the quotePreference for Typescript to single?

### DISCUSSION THREAD

**Discussion by LE0Ntt** [zed-industries/zed#7917] [Feature Requests]
I did this:
```json
"lsp": {
    "typescript-language-server": {
      "initialization_options": {
        "preferences": {
          "quotePreference": "single"
        }
      }
    }
  }
```
but it sadly doesn't work, so is there another way?

**Comment by SomeoneToIgnore** [comment]
Is that the formatting-related setting?

Consider https://github.com/zed-industries/zed/discussions/7951#discussioncomment-8501391 explanation on how formatting works, most probably you have to configure prettier (or adjust the formatting options to use the language server instead of prettier).

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
